src/encoded/upgrade/donor.py

```diff
@@ -226,7 +226,7 @@ def fly_donor_9_10(value, system):
         for alias in value['aliases']:
             if alias[0:6] == 'kyoto:':
                 new_alias = alias.replace(
-                    'kyoto:', 'dggr:', 1)
+                    'kyoto:', 'encode:', 1)
```

Comment:
I'd say do `encode:kyoto_` as Idan suggested so that he can later on check the demo and see if there is really nothing like that.